[ID1000725] add aliyun service configuration document

Nixiangyu opened the pull request:
--story=1000725 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000725

Signed-off-by: Nixiangyu nixiangyu@ofworld.com

Commit:
[ID1000725] add aliyun service configuration document

--story=1000725 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000725

Signed-off-by: Nixiangyu nixiangyu@ofworld.com

Commit:
[ID1000725] add aliyun service configuration document

--story=1000725 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000725

Signed-off-by: Nixiangyu nixiangyu@ofworld.com

Commit:
[ID1000725] add aliyun service configuration document

--story=1000725 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000725

Signed-off-by: Nixiangyu nixiangyu@ofworld.com

zhangbing16 commented on sampleconfig/donation-service.yaml:
```diff
@@ -46,3 +46,15 @@ Database:
+#
+################################################################################
+ALiYun:
+    Endpoint: oss-cn-beijing.aliyuncs.com
```
http://oss-cn-beijing.aliyuncs.com?

Nixiangyu replied:
是的，后期oss的框架的自带方法会处理这个endpoint，那我加上https吧

zhangbing16 commented on sampleconfig/donation-service.yaml:
```diff
@@ -46,3 +46,15 @@ Database:
+    Endpoint: oss-cn-beijing.aliyuncs.com
+    AccessKeyID: test
+    AccessKeySecret: test
+    NewsFeedImage: donation-oss
```
这个是bucketName吗？

Nixiangyu replied:
那我换成bucketName吧

Commit:
[ID1000725] add aliyun service configuration document

--story=1000725 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000725

Signed-off-by: Nixiangyu nixiangyu@ofworld.com
